src/audio-worklet/free-queue/example/js/script.js

```diff
@@ -0,0 +1,71 @@
+// ***** Tests *****
+
+let secureContext = window.isSecureContext;
+let crossOriginIsolated = window.crossOriginIsolated;
+let SharedArrayBufferAvailable = window.SharedArrayBuffer !== undefined
+let AudioWorkletAvailable = window.AudioWorklet !== undefined
+let WebAssemblyAvailable = window.WebAssembly !== undefined
+
+
+const toogle_button = document.getElementById('toogle');
+const checksTable = document.getElementById('checks');
+
+let checks = [
+    ["Secure Context", secureContext],
+    ["Cross Origin Isolated", crossOriginIsolated],
+    ["SharedArrayBuffer", SharedArrayBufferAvailable],
+    ["AudioWorklet", AudioWorkletAvailable],
+    ["WebAssembly", WebAssemblyAvailable]
+]
+
+checks.forEach(check => {
+    checksTable.innerHTML += `
+        <tr>
+            <td>${check[0]}</td>
+            <td>${check[1] ? "✔️" : "❌" }</td>
+        </tr>
+    `
+})
+
+if (checks.every(check => check[1])) {
+    toogle_button.disabled = false;
+}
+
+// *****************
```

Comment:
This type of comments is not helpful. What's your purpose for this comment? Perhaps it's better to write an overview of the following section?